cypress/integration/check_links.js

```diff
@@ -29,7 +29,8 @@ context("Check for broken links", () => {
                   '/en/science/'
                 ]
     const allowlist = [
-      'https://testbuchen.de/#/?zoom=0&lat=47.71401323721353&lng=8.66960999999999'
+      'https://testbuchen.de/#/?zoom=0&lat=47.71401323721353&lng=8.66960999999999',
+      'https://onlinelibrary.wiley.com/doi/abs/10.2307/3315826.n'
```

Comment:
```suggestion
      'https://onlinelibrary.wiley.com/doi/abs/10.2307/3315826.n1'
```

The URL was incomplete. 

- See original issue https://github.com/corona-warn-app/cwa-website/issues/2490